libcxx/include/support/atomic/atomic_cuda.h

```diff
@@ -97,6 +97,8 @@ namespace __host {
 #elif defined (_LIBCUDACXX_HAS_C11_ATOMIC_IMP)
 //TODO
 // #  include "atomic_c11.h"
+#elif defined(_LIBCUDACXX_COMPILER_NVRTC)
+#  include "atomic_nvrtc.h"
```

Comment:
Ooooh. It's because prior to the refactor, all the host bits were ifdef'd out, but now we depend on storage types and whatnot from `__host`? Makes sense.

Reply:
Right, we should make an abstraction for storage. I remember attempting to do that and finding it too difficult to remove the scope parameter in the same change.